crates/ruff/src/rules/flake8_simplify/rules/ast_if.rs

```diff
@@ -81,6 +82,40 @@ impl Violation for NeedlessBool {
     }
 }
 
+define_violation!(
+    /// ### What it does
+    /// Checks for three or more consective if-statements with direct returns
+    ///
+    /// ### Why is this bad?
+    /// These can be simplified by using a dictionary
+    ///
+    /// ### Example
+    /// ```python
+    /// if x = 1:
+    ///     return "Hello"
+    /// elif x = 2:
+    ///     return "Goodbye"
+    /// else:
+    ///    return "Goodnight"
+    /// ```
+    ///
+    /// Use instead:
+    /// ```python
+    /// return {1: "Hello", 2: "Goodbye"}.get(x, "Goodnight")
+    /// ```
+    pub struct IfToDict;
+);
+impl AlwaysAutofixableViolation for IfToDict {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Use a dictionary instead of consecutive `if` statements")
```

Comment:
Nit: You can directly use `String::new("Use...")` because you're not using any formatting arguments.

Reply:
Though it's not obvious, the use of `format!` here is significant as we pick up the `format!` calls and use them to generate documentation on the possible message formats (see: `#[derive_message_formats]`).